src/test/java/net/syscon/elite/service/CourtHearingsServiceTest.java

```diff
@@ -258,6 +264,38 @@ private void givenCourtNotFound() {
         when(agencyLocationRepository.findById("COURT")).thenReturn(Optional.empty());
     }
 
+    @Test
+    void scheduleHearing_errors_when_court_is_not_currently_active() {
+        givenProvidedCourtIsNotActive();
+
+        assertThatThrownBy(() -> courtHearingsService.scheduleHearing(OFFENDER_BOOKING_ID, PRISON_TO_COURT_HEARING))
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessage("Supplied court location wih id %s is not active.", PRISON_TO_COURT_HEARING.getToCourtLocation());
```

Comment:
not a great fan of testing error messages with dynamic strings in them, especially as text exactly the same as the java.  Would be better to just write
```java
                .hasMessage("Supplied court location wih id COURT is not active.")
```
especially since `COURT` is then used explicitly in the mockito when clause anyway